web/static.go

```diff
@@ -82,11 +82,7 @@ func staticHandler(handler http.Handler) http.Handler {
 
 func pluginHandler(config model.ConfigFunc, handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if *config().ServiceSettings.EnableDeveloper {
-			w.Header().Set("Cache-Control", "no-cache, no-store, must-revalidate")
-		} else {
-			w.Header().Set("Cache-Control", "max-age=31556926, public")
-		}
```

Comment:
Can we still not cache at all in developer mode? One of the methods I use to quickly iterate when developing the client-side of plugins is to manually build and copy the webapp bundle into the static dir, and that won't cause the md5 hash to update

Reply:
Two thoughts:
1) Can you disable the browser cache via the developer tools instead? I find this a more holistic solution to caching problems in general vs. having special server behaviour in developer mode.
2) Will the new `make deploy` help with this? (https://github.com/mattermost/mattermost-plugin-sample/blob/master/Makefile#L86-L102) I now run `make dist deploy` and the existing plugin is replaced via the API in a matter of seconds, with no manual file copying required. It's almost "hot reloading" for plugins.

Reply:
(If we do keep thing as proposed, minimally I'll get rid of `pluginHandler` in favour of the now identical `staticHandler`. I didn't look closely enough to realize they were identical.)